README.md

```diff
@@ -49,7 +49,19 @@ The media type for JSON Siren is `application/vnd.siren+json`.
       "fields": [
         { "name": "orderNumber", "type": "hidden", "value": "42" },
         { "name": "productCode", "type": "text" },
-        { "name": "quantity", "type": "number" }
+        { "name": "quantity", "type": "number" },
+        { "name": "color", "type": "radio", "title": "Choose a Color", 
```

Comment:
A name describing the control. **Field names MUST be unique within the set of fields for an action.** The behaviour of clients when parsing a Siren document that violates this constraint is undefined. Required.